### PR TITLE
Ops: automated packaging

### PR DESCRIPTION
**Pull request by mrbusysky**
**New and improved:**

- Working release process

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      if: env.minor_change == 'true'
+      run: |
+        zip -r Scenario3D-v${{ env.version }}.zip Scenario README.md Scenario3DPackageInstaller.cs Scenario3D.wlt ScenarioPackageInstaller2.cs
+
+    - name: Create Release
+      id: create_release
+      if: env.minor_change == 'true'
+      uses: actions/create-release@v1
+      env: 
+        GITHUB_TOKEN: ${{ secrets.NOT_GITHUB_TOKEN }}
```
[the remark this answers is not in the document]
Comment (mrbusysky) [reply .github/workflows/release.yml:44]: Because github refuses you to name it GITHUB_TOKEN

[the remark this answers is not in the document]
Comment (mrbusysky) [reply .github/workflows/release.yml:44]: I just chose whatever. It's up to you guys what you want to name it

[the remark this answers is not in the document]
Comment (qvaleroo) [reply .github/workflows/release.yml:44]: `NOT` is confusing 😄  You can name it `CI_GITHUB_TOKEN`

[the remark this answers is not in the document]
Comment (mrbusysky) [reply .github/workflows/release.yml]: Updated

**Review by hervenivon — APPROVED** [review]

**Pull request merged**